front/src/common/SpeedLimitByTagSelector/SpeedLimitByTagSelector.tsx

```diff
@@ -48,7 +48,6 @@ export default function SpeedLimitByTagSelector({
           id="speed-limit-by-tag-selector"
           value={speedLimitByTag || ''}
           label={t('speedLimitByTagAbbrev')}
-          placeholder={t('noSpeedLimitByTag')}
```

Comment:
Why do you remove this placeholder ?

Reply:
It's a mistake, I removed it without realizing it also affected operational studies. I have put it back.